test/functional/corporate_information_pages_controller_test.rb

```diff
@@ -99,7 +101,7 @@ class CorporateInformationPagesControllerTest < ActionController::TestCase
   test "finds unpublishing for a corporate information page" do
     organisation = create(:organisation)
     cip = create(:corporate_information_page, :unpublished, organisation:)
-    alternative_url = Whitehall.url_maker.root_url
+    alternative_url = "https://www.gov.uk"
```

Comment:
Is there a helper method available that returns this string, to avoid duplicating it here?

Reply:
Yes, will update to use Plek's `website_root` method.